tests/php/Forms/FormSchemaTest.php

```diff
@@ -22,43 +21,7 @@ public function testGetSchema()
     {
         $form = new Form(null, 'TestForm', new FieldList(), new FieldList());
         $formSchema = new FormSchema();
-        $expected = [
-            'name' => 'TestForm',
-            'id' => 'Form_TestForm',
-            'action' => null,
-            'method' => 'POST',
-            'attributes' => [
-                'id' => 'Form_TestForm',
-                'action' => null,
-                'method' => 'POST',
-                'enctype' => 'application/x-www-form-urlencoded',
-                'target' => null,
-                'class' => ''
-            ],
-            'data' => [],
-            'fields' => [
-                [
-                    'id' => 'Form_TestForm_SecurityID',
-                    'name' => 'SecurityID',
-                    'type' => "Hidden",
-                    'component' => null,
-                    'holderId' => 'Form_TestForm_SecurityID_Holder',
-                    'title' => 'Security ID',
-                    'source' => null,
-                    'extraClass' => 'hidden',
-                    'description' => null,
-                    'rightTitle' => null,
-                    'leftTitle' => null,
-                    'readOnly' => false,
-                    'disabled' => false,
-                    'customValidationMessage' => '',
-                    'attributes' => [],
-                    'data' => [],
-                    'validation' => [],
-                ],
-            ],
-            'actions' => []
-        ];
+        $expected = json_decode(file_get_contents(__DIR__.'/FormSchemaTest/testGetSchema.json'), true);
```

Comment:
maybe remove the `json_decode()` and the `json_encode($expected)` below?
encoding for the sake of decoding :D